### ISSUE TITLE
it's impossible to paste source code in it - it formats it automatically

### ISSUE THREAD

**Issue by Zig-03** [gobengo/distbin#31]

**Comment by gobengo** [comment]
Distbin renders input text as markdown.

You can use that to make code blocks by surrounding with triple backticks

Syntax examples: https://help.github.com/en/articles/creating-and-highlighting-code-blocks#fenced-code-blocks

Example post on Distbin.com
http://distbin.com/activities/e965af20-d520-4dbf-953a-35946933ee8c

**Comment by Zig-03** [comment]
> Does this help with what you’re trying to do?

Yes, thank you, but why is so complicated? I'll get tired typing all those things, especially if I wanna use distbin exclusively for sharing code.
I'd suggest adding buttons, like on github, there are buttons for: bold, italic, **code**, quotes, links...
